docker/dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:18.04
+
+COPY *.deb /
+
+RUN apt update && \
+  apt install -y curl wget && \
+  apt install -y /*.deb && \
+  rm -rf /*.deb /var/lib/apt/lists/*
```

Comment:
Are we OK with a docker directory in the main repo or do we want this contained in the `.cicd` directory with other build related scripts and Dockerfiles @kj4ezj ?

Reply:
I think we want "official" docker images to be outside of the `.cicd` folder, even though they're build by the CI system, for visibility to the community. This is where our old "offical" dockerfiles used to live.